ui/app/rif/components/tabs/index.js

```diff
@@ -10,6 +10,8 @@ class Tabs extends Component {
     onChange: PropTypes.func,
     showBack: PropTypes.bool,
     backAction: PropTypes.func,
+    classes: PropTypes.any,
+    letSelectedIndex: PropTypes.number,
```

Comment:
Why is this prop needed?

Reply:
I deleted, cause i didnt notice that we can set this in taboptions